lib/config/options/index.ts

```diff
@@ -3097,6 +3097,45 @@ const options: RenovateOptions[] = [
     default: 90,
     globalOnly: true,
   },
+  {
+    name: 'deleteConfigFile',
+    description:
+      'If set to `true`, Renovate tries to delete the self-hosted config file after reading it.',
+    type: 'boolean',
+    default: false,
+    globalOnly: true,
+  },
+  {
+    name: 'eagerGlobalExtends',
```

Comment:
```suggestion
    name: 'globalExtendsEager',
```

Let's rename it so that it sorts next to globalExtends